src/inspector/view/qml/MuseScore/Inspector/general/appearance/internal/ArrangeSection.qml

```diff
@@ -46,10 +51,27 @@ Column {
 
         spacing: 8
 
-        StyledTextLabel {
+        Item {
+            height: childrenRect.height
             width: parent.width
-            text: qsTrc("inspector", "Arrange")
-            horizontalAlignment: Text.AlignLeft
+
+            StyledTextLabel {
+                width: parent.width
+                text: qsTrc("inspector", "Arrange")
+                horizontalAlignment: Text.AlignLeft
+            }
+
+            PropertyResetButton {
+                anchors.top: parent.top
+                anchors.right: parent.right
+
+                navigation.name: root.navigationName + "Reset"
+                navigation.panel: root.navigationPanel
+                navigation.row: root.buttonNavigationRow
+                navigation.accessible.name: qsTrc("inspector", "Reset order to default value")
```

Comment:
Maybe "Reset _arrangement_ to default" might be clearer, but I'm not sure

Reply:
@cbjeukendrup I was wondering about this actually - I think in MS3 it was called "stacking order" which might be an option?

Reply:
Ah, yes, that might be the best candidate actually!